### PR TITLE
Add links and spell out acronyms

### PR DESCRIPTION
**Pull request by appellj**
Add links for easy navigation to references. To adhere to the style guide, spell out acronyms on first use. Also add a serial comma to make the first sentence easier to parse.

### CODE REVIEW

File: TOB.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Marsh Gardiner @earth2marsh
 
 Ron Ratovsky @webron
 
-## BGB Elected - terms through May 2020
+## Business Governance Board Elected - terms through May 2020
 
 Darrel Miller @darrelmiller
```
Comment (darrelmiller) [inline TOB.md:17]:
```suggestion
Darrel Miller @darrel_miller
```

**Comment by darrelmiller** [comment]
@ncaidin Is this true and my TOB term is up?  Do we need to have a new vote?

**Comment by ncaidin** [comment]
Yes @darrelmiller . I meant to hold the elections earlier in the year but it somehow got by me. I will send out a plan for the election to the Business Governance Board this week, and we will hold the election over the next 2 - 3 weeks. Three people from the TOB are up for election by the BGB (last year it was election by the TSC maintainers).

**Review by lornajane — APPROVED** [review]
Useful and inclusive! +1 from me

**Comment by MikeRalphson** [comment]
Please could you resolve the merge conflicts?

**Comment by lornajane** [comment]
Closing due to inactivity (feel free to re-open with the requested updates)

**Pull request closed without merging**